src/Raven.Pal/pal.ver

```diff
@@ -1 +1 @@
-62022
+62040
```

Comment:
let's maybe go to 71000, in case we would want to make any changes in the 6.2 PAL

Reply:
This is a partial PR just for review. In the PR for the `ravendb/ravendb` this is `71000` already